### ISSUE TITLE
Usability feedback: Comma Separator

### ISSUE THREAD

**Issue by Sparrow0hawk** [acteng/update-your-capital-schemes#138]
# Usability feedback

## Comma Separator (Usability)

Monetary input is insufficiently tolerant of legitimate user input, against GOV.UK Design System guidance.

### Reference:
Usability

## Issue ID: DAC_Comma_Separator_Usability_01

URL: https://test.update-your-capital-schemes.activetravelengland.gov.uk/schemes/64/spend-to-date
Page title: 'Error: Update your capital schemes - Active Travel England - GOV.UK'
H1 heading: 'Change spend to date'
Journey: 1.2g

Screenshot:
![image](https://github.com/user-attachments/assets/944cdbcf-8d9a-4c98-9af7-33bcb0544e8d)

On the 'Change spend to date' page, the 'how much has been spent to date?' input does not accept a comma separator for larger numbers.

Whilst this affects all users, this can be a particular problem for voice activation users, as the Dragon voice activation software will typically render the dictation of 'seven thousand five hundred' as '7,500', including comma separators, by default.

It is possible for Dragon users to then issue a correction command and select '7500' from a list of alternatives, or they can dictate the individual digits, but these involve performing extra steps and make input slower.

### Current code ref(s):
#amount
```html
<input class="govuk-input govuk-input--width-10 govuk-input--error" id="amount" name="amount" type="text" spellcheck="false" value="751,000" aria-describedby="amount-error" inputmode="numeric" required="required">
```

### Voice activation user comments:

> “When I selected the ‘confirm’ button after dictating into the edit field an error message appeared. The error message seemed to indicate that it is not a number that has been inputted into the field, but this is not the case. As a result of this I was not sure what I needed to do to correct the issue.”

### Examples of additional instances:

Additional instances of this issue may exist on other pages throughout the website; wherever this issue occurs, they too will need to be resolved.

### Solution:

Generally, the service should be more tolerant of any legitimate user input. Particularly in light of the service itself consistently displaying monetary values in a format such as '£7,500' or '£1,200,000' that it does not then accept as user input, setting a potentially misleading example.

More specifically, for voice activation users dictating values into a text input, the Dragon voice activation software will, by default, include comma separators and the currency prefix, if the user naturally says 'pounds', and while this can be corrected, or the digits spoken individually to ensure correct entry, this requires performing extra steps.

As detailed by [GOV.UK Design System: Text input (Prefixes and suffixes)](https://design-system.service.gov.uk/components/text-input/#prefixes-and-suffixes), the addition of the currency symbol should be gracefully tolerated and, similarly, comma separators can be equally accepted, as legitimate alternative forms of user input that do not affect the meaning of their entry.
